HelpSource/Guides/Internal-Snooping.schelp

```diff
@@ -57,38 +66,39 @@ Server.filenameSymbol.postln; // the path to the file that defined this class
 
 (
 // print all classes whose names start with 'F'
-Class.allClasses.do({ arg class;
-	if (class.name.asString.at(0) == $F, { class.name.postln; });
-})
+Class.allClasses.do { | class |
```

Comment:
>but I think it's a little inconsiderate to start applying your syntax style preferences to code examples without a discussion beforehand. 

Over time, this syntax has become the most common one. Let's not be too picky, it should be fun, no?